docs/src/components/HeaderBox.tsx

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from "styled-components";
+import { mediaQueries as mq } from "@kiwicom/orbit-components";
+
+import { MAX_CONTENT_WIDTH } from "../consts";
+
+const HeaderBox = styled.div`
+  ${mq.desktop(css`
+    position: relative;
+    height: 100vh;
+    padding: 64px;
+    background: linear-gradient(80.68deg, #f1f4f7 7.05%, #f5f7f9 57.05%);
+    z-index: 0;
+  `)}
+
+  display: block;
+  border-radius: 24px;
+  border-bottom-right-radius: 50%;
+  width: 100%;
+  max-width: ${MAX_CONTENT_WIDTH};
```

Comment:
I would not use this constant here. Just set it as 90rem. It will not be probably used anywhere else.

Reply:
Hmm I don't think I can. Because the TopBar and the footer are supposed to have the same width as the HeaderBox. The content is the one that is actually smaller, hence the change. The content stays at 80rem while everything else grows to 90 rem. WDYT?